resources/js/components/modules/data/alcoholModule.js

```diff
@@ -525,8 +525,16 @@ export const alcoholModuleSlides = {
       }
     }
   },
+  // slide  32
+  17:{
+    slide_type: 'emailForm',
+    header:{
+      title:null,
```

Comment:
The template needs to grab the title ***`title: 'Create your own wellness goal!'`*** from the JSON instead of being hard-coded in the template. We are unsure if this template is going to be reused again so we need to take steps to ensure it will be easy to change from the JSON.